src/components/panel/panel.component.tsx

```diff
@@ -0,0 +1,92 @@
+import { Button } from "src/components/button";
+import { Expand } from "src/components/icons/expand.component";
+import { Pill } from "src/components/pill";
+import { Tabs } from "src/components/tabs/tabs.component";
+import { useGetObject } from "src/hooks/useGetObject";
+
+interface Props {
+  objectType: string;
+  togglePanel: () => void;
+  uid: string;
+}
+
+export const Panel = ({ togglePanel, objectType, uid }: Props) => {
+  const { data } = useGetObject(objectType, {
+    uid: uid,
+  });
+
+  const getTitle = (object: { [k: string]: string }, priority: string[]) => {
+    const [title] = priority.map((key) => object[key]).filter((key) => key);
+
+    return title;
+  };
+
+  const orderedKeys = ["title", "name", "uid"];
+
+  return (
+    <div className="z-50">
```

Comment:
make this first `div` the fixed part, then I think. you should be able to remove `fixed` in the other two places

There is a jump on the page of the content behind moving down when I open

Reply:
it doesn't work with that change. can't see the jump either